### ISSUE TITLE
Create ARM builds in Launchpad PPA

### ISSUE THREAD

**Issue by apandada1** [JoseExposito/touchegg#522]
### Is your feature request related to a problem? Please describe

Right now the Launchpad PPA only hosts packages for amd64 architecture. Please enable builds for `armhf` and `arm64`. People are [trying to use](https://askubuntu.com/questions/1362903/unable-to-convince-apt-to-use-the-ppa-version-of-a-package-rather-than-the-unive) the PPA on Raspberry Pi, but it does not work.

### Describe the solution you'd like

[Here](https://askubuntu.com/questions/371339/how-to-create-ppa-packages-for-arm-or-powerpc-in-launchpad/765149#765149) is how to enable ARM builds. You need to enable it only once, and later on, Launchpad would automatically build for arm just as it does for amd64.

I tried building the package for ARM in Launchpad, and it builds successfully. https://launchpad.net/~apandada1/+archive/ubuntu/expt/+build/22062995

### Describe alternatives you've considered

There are .deb packages for ARM in Launchpad, but users cannot get automatic updates with that.

**Comment by apandada1** [comment]
Thank you for the quick fix.

**Comment by JoseExposito** [comment]
No problem :) I didn't have time to comment here, but I hope the builds are working. Feel free to create a new issue if you find problems with the installation.
